news/_posts/public-oss-demo/2021-03-16-regular-public-oss-demo-schedule.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "PrestaShop Public OSS Demo - Choosing the pace"
+subtitle: "We aim to host open source project's public demo every month"
+date:   2021-03-16 10:00:00
+authors: [ MathieuFerment ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+Following [the February session](https://build.prestashop.com/news/public-oss-demo-2/) of the open source project's public demo, we now feel confident to set a pace for these events.
+
+## PrestaShop open source project's public demo every month
+
+We aim to host open source project's public demos the last Wednesday of each month at 2pm CET.
+
+That means that the March session will happen on [**Wednesday 31st of March 2021, 2pm CET**](https://www.youtube.com/watch?v=UJ_XQs8trs4).
+
+The session will be kept in record on YouTube, so it can be watched later if this timeslot is not suitable for you.
+
+
+## Sessions agenda
+
+For now we stick to the current content of these sessions, which means in each session we
+- present the event
+- present latest announcements for the project
+- present some of the topics achieved this month
+- host a Q&A session
+
+By the way, the Q&A session does not need to be about the demonstrations content! It can be any question related to the open source project, be it the scope, the documentation, the process or the content of the project.
```

Comment:
```suggestion
By the way, the Q&A session does not need to be about the demonstrations content! It can be any question related to the open source project, the scope, the documentation, the process or the content of the project.
```

Reply:
I think it make sense to have the "be it" here 👌